album/references-beyond-homestuck.yaml

```diff
@@ -296,6 +296,129 @@ URLs:
 ---
+Track: Celestial
+Directory: celestial-remix
+Always Reference By Directory: true
+Date First Released: December 14, 2023
+Duration: '4:42'
+Artists:
+- Pokémon
+- Toby Fox (Remixer)
```

Comment:
For other remixes, we don't list a role for the remixer, do we?

Reply:
the credits explicitly say "Remixed by Toby Fox", I put (remixer) to reflect that credited role. feel free to remove if it's unnecessary^^